pkgs/development/compilers/idris2/default.nix

```diff
@@ -91,6 +92,6 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/idris-lang/Idris2";
     license = lib.licenses.bsd3;
     maintainers = with lib.maintainers; [ fabianhjr wchresta ];
-    inherit (chez.meta) platforms;
+    inherit (chez.meta) platforms badPlatforms;
```

Comment:
Why inherit chez's badPlatforms?

Maybe (chez + chezRacket) platforms?

Reply:
The reasoning was that we work whenever chez works. If chez breaks, we break. But things with chez vs chez-racket complicate things, so I'll leave it as we have it today for now.